libraries/botbuilder-ai/src/qnaMakerDialog.ts

```diff
@@ -607,17 +699,24 @@ export class QnAMakerDialog extends WaterfallDialog implements QnAMakerDialogCon
             qnaResponse.answers = qna.getLowScoreVariation(qnaResponse.answers);
 
             if (isActiveLearningEnabled && qnaResponse.answers?.length > 1) {
-                const suggestedQuestions: string[] = [];
-
-                qnaResponse.answers.forEach((answer) => {
-                    suggestedQuestions.push(answer.questions[0]);
-                });
-
-                const message = QnACardBuilder.getSuggestionsCard(
-                    suggestedQuestions,
-                    dialogOptions.qnaDialogResponseOptions.activeLearningCardTitle,
-                    dialogOptions.qnaDialogResponseOptions.cardNoMatchText
-                );
+                const suggestedQuestions = qnaResponse.answers.map((answer) => answer.questions[0]);
+
+                const message = this.suggestionsActivityFactory
```

Comment:
I _think_ you can write this is `const message = this.suggestionsActivityFactory?.(...) ?? QnACardBuilder.getSuggestionsCard(...);`